util_inference_example.py

```diff
@@ -0,0 +1,87 @@
+"""
+Usage: python util_inference_example.py image annotations
+
+Adapted from the dense_inference.py to demonstate the usage of the util
+functions.
+"""
+
+import sys
+import numpy as np
+import cv2
+import densecrf as dcrf
+import matplotlib.pylab as plt
+from skimage.segmentation import relabel_sequential
+
+from utils import compute_unary, create_pairwise_bilateral, \
+    create_pairwise_gaussian
```

Comment:
We still need to make it installable and usable as `from densecrf.utils import compute_unary`. If you know how to do so, you can update this PR, otherwise I'll look into it later.